build.sbt

```diff
@@ -312,15 +335,36 @@ lazy val stagedRelease = (project in file("core/src/test"))
 
 import ReleaseTransformations._
 
-// Don't use sbt-release's cross facility	
+// Don't use sbt-release's cross facility
 releaseCrossBuild := false
 
-def crossReleaseStep(step: ReleaseStep): Seq[ReleaseStep] = {
+lazy val downVersionPySpark = taskKey[Unit]("Replace PySpark")
+downVersionPySpark := {
+  "./downversion-pyspark.sh" !
+}
+
+lazy val updateCondaEnv = taskKey[Unit]("Update Glow Env To Latest Version")
+updateCondaEnv := {
+  "conda env update -f python/environment.yml" !
+}
+
+def crossReleaseStep(step: ReleaseStep, test: Boolean): Seq[ReleaseStep] = {
+  val updateCondaEnvStep = releaseStepCommandAndRemaining(if (test) "updateCondaEnv" else "")
+  val downVersionPySparkStep = releaseStepCommandAndRemaining(
+    if (test) "downVersionPySpark" else "")
```

Comment:
I assume and my tests verify that pyspark is only needed at the ``runTest`` stage and not at ``publishArtifacts`` or ``stagedRelease/test``. So I set pyspark version changes to be applied only for ``runTest``. Let me know if this is not true.

Reply:
I agree they won't be needed for `publishArtifacts` and `stagedRelease` - both only touch the Scala side.

Reply:
Great